src/main/kotlin/io/cloudflight/gradle/autoconfigure/kotlin/KotlinConfigurePlugin.kt

```diff
@@ -0,0 +1,88 @@
+package io.cloudflight.gradle.autoconfigure.kotlin
+
+import io.cloudflight.gradle.autoconfigure.extentions.gradle.api.plugins.apply
+import io.cloudflight.gradle.autoconfigure.extentions.gradle.api.plugins.create
+import io.cloudflight.gradle.autoconfigure.extentions.gradle.api.plugins.getByType
+import io.cloudflight.gradle.autoconfigure.java.JavaConfigurePlugin
+import io.cloudflight.gradle.autoconfigure.java.JavaConfigurePluginExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.jvm.toolchain.JavaToolchainSpec
+import org.jetbrains.kotlin.allopen.gradle.AllOpenExtension
+import org.jetbrains.kotlin.allopen.gradle.SpringGradleSubplugin
+import org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension
+import org.jetbrains.kotlin.gradle.internal.Kapt3GradleSubplugin
+import org.jetbrains.kotlin.gradle.plugin.KotlinPluginWrapper
+import org.jetbrains.kotlin.gradle.plugin.getKotlinPluginVersion
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.kotlin.noarg.gradle.KotlinJpaSubplugin
+import org.slf4j.LoggerFactory
+
+class KotlinConfigurePlugin : Plugin<Project> {
+    override fun apply(project: Project) {
+        project.plugins.apply(JavaConfigurePlugin::class)
```

Comment:
what happens if the plugin is applied twice?   
will it just not run the second time?

asking because you first apply the JavaConfigurePlugin in the AutoConfigurePlugin which creates the extension and then apply it here again which would create the extension again and that would fail.